src/core/index.ts

```diff
@@ -57,13 +57,26 @@ export const journey = wrapFnWithLocation(
   }
 );
 
-export const step = wrapFnWithLocation(
+type StepType = (name: string, callback: VoidCallback) => Step;
+
+interface StepWithSkipType extends StepType {
+  skip?: (name: string, callback: VoidCallback) => Step;
+}
+
+export const step: StepWithSkipType = wrapFnWithLocation(
   (location: Location, name: string, callback: VoidCallback) => {
     log(`Step register: ${name}`);
     return runner.currentJourney?.addStep(name, callback, location);
   }
 );
 
+step.skip = wrapFnWithLocation(
```

Comment:
If we are going to resemble PW, it also has support for skipped based on a callback based function. 

```ts
step.skip((env)=> env === "development", 'Skipping dev tests')
```

It would be nice to add support for this as well.

Reply:
let's do that in a follow up